app/services/classification/base.rb

```diff
@@ -53,11 +53,12 @@ def values_hash(options = {})
         end
         value = nil
         if key.merge_overrides
+          default = key.default_value if key.merge_default
```

Comment:
I'd prefer being explicit here, this works since ruby saw the variable but looks like `default` might be uninitialized local variable after first look since it's hidden by if without else

```ruby
default = key.merge_default ? key.default_value : nil
```